<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataConfigDialog</class>
 <widget class="QDialog" name="DataConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>数据库状态：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="DataBaseStatu_label">
                 <property name="text">
                  <string>未连接</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>设备Mac:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Mac_comboBox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>设备名称：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="Name_comboBox"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>输入Mac查询:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="Mac_Input_lineEdit">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>185</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="Input_Query_pushButton">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>185</height>
                </size>
               </property>
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="ConnectDB_pushButton">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>185</height>
              </size>
             </property>
             <property name="text">
              <string>连接数据库查询</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>最大显示数据数量：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="MaxItemCount_spinBox">
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>500000</number>
           </property>
           <property name="singleStep">
            <number>500</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="Data_tableWidget"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="Accept_pushButton">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Cancel_pushButton">
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>曲线</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="DrawOnchart_pushButton">
              <property name="text">
               <string>曲线图</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>起始:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="IndexStart_spinBox">
                <property name="maximum">
                 <number>500000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>结束:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="IndexEnd_spinBox">
                <property name="maximum">
                 <number>500000</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QtCharts::QChartView" name="ChartView"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="Message_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
